update-prs-auto-merge-enabled/script.js

```
@@ -0,0 +1,57 @@
+
+    const {repository: {pullRequests: {nodes: pullRequestNodes}}} = await github.graphql(query, {
+        owner: context.repo.owner,
+        repo: context.repo.repo,
+        baseRef: BASE_BRANCH,
+    });
+
+    if (!context.payload.pull_request?.merged) {
+    }
+
+    const autoMergeEnabledPRs = pullRequestNodes
+        .filter(pr => pr.autoMergeRequest && pr.autoMergeRequest.enabledAt && pr.mergeable === 'MERGEABLE')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)